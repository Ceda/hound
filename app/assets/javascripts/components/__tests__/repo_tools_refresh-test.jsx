import React from 'react';
import renderer from 'react-test-renderer';

import RepoToolsRefresh from '../repo_tools_refresh.js';

const Hound = window.Hound = global.Hound = {
  settings: {
    placeholder: "meh"
  }
};

it('renders appropriately', () => {
  const onRefreshClicked = jest.genMockFunction();

  const component = renderer.create(
    <RepoToolsRefresh
      isSyncing={false}
      onRefreshClicked={onRefreshClicked}
    />
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
