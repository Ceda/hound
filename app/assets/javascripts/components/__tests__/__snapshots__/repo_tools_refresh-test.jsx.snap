exports[`test renders appropriately 1`] = `
<div
  className="repo-tools-refresh">
  <button
    className="repo-tools-refresh-button"
    disabled={null}
    onClick={[Function]}>
    <span />
  </button>
</div>
`;
